### ISSUE TITLE
Typos and broken links in the Index page

### ISSUE THREAD

**Issue by yadunandanbhat** [apache/openwhisk-website#495]
1. Typo: 
        "Work with what you know and love. OpenWhisk supports a growing list of your favorite languages such as Go, Java, Rust and even newer, cloud-native ones **_like There is_** even an experimental runtime for Deno in-development."

2. Broken Links:
        Weather's link is 404 (See https://github.com/apache/openwhisk-catalog/tree/master/packages/weather)
        RSS and Jira are deprecated (See https://github.com/apache/openwhisk-package-rss & https://github.com/apache/openwhisk-package-jira)

Can I update this and open a PR?

**Comment by style95** [comment]
Thanks, @yadunandanbhat
A contribution is always welcome.

Though I am unclear the new sentence describes the intention better.

**Comment by yadunandanbhat** [comment]
No, that's not the new sentence, the highlighted part is the error. I think  "...cloud-native ones. There is also an experimental runtime..." would be better

**Comment by style95** [comment]
oh, I got it.
